Utilities/MCStepLogger/src/StepInfo.h

```diff
@@ -143,8 +143,8 @@ struct StepInfo {
   float y = 0.;
   float z = 0.;
   float E = 0.;
-  float step = 0.;
-  float maxstep = 0.;
+  double step = 0.;
+  double maxstep = 0.;
```

Comment:
not sure we need this. I doubt that we need to distinguish steps below the float resolution. This will also increase the file size / memory requirement quite a lot.

Reply:
will check that.